releasenotes/notes/acceptance-region-9431491e16222211.yaml

```diff
@@ -0,0 +1,4 @@
+---
+other:
+  - |
+    **[Acceptance]** Set ``Region`` while client is generating (`#1298 <https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/1298>`_)
```

Comment:
The change affects not only acceptance tests. Better to say that you read effective region from the IAM. Also that doesn't seem to affect user experience, so I don't think that RN is required here

Reply:
done